### ISSUE TITLE
Pin SFS-Python version

### ISSUE THREAD

**Issue by mgeier** [sfstoolbox/sfs-documentation#32]
Currently, only `sfs` is required.

Since the API of SFS-Python is likely to change from time to time, it would be good to require a specific version to make sure all code examples actually work.

**Comment by hagenw** [comment]
Good point, I will adjust this before the DAGA. I just have to check if I want to go with `sfs` 0.4 or 0.5

**Comment by mgeier** [comment]
You could start by pinning `sfs==0.4.*` now, because without that, the build will break with the upcoming release of SFS 0.5.0.

At some later point, you could update the code examples and change the required version to `sfs==0.5.*`.

**Comment by hagenw** [comment]
I updated the API to the upcoming `sfs` and pinned the version to `sfs>=0.5.0`
